locator/tablets.hh

```diff
@@ -302,6 +336,15 @@ struct load_stats {
     }
 };
 
+struct repair_scheduler_config {
+    // When set to false, auto repair is disabled
```

Comment:
nit: this comment doesn't add anything beyond what the member name already indicates

Reply:
I will drop it then.